lib/puppet/type/cs_clone.rb

```diff
@@ -18,42 +18,62 @@
     desc 'The corosync resource primitive to be cloned.'
   end
 
+  newproperty(:group) do
+    desc 'The corosync resource group to be cloned.'
+  end
+
   newproperty(:clone_max) do
     desc "How many copies of the resource to start.
       Defaults to the number of nodes in the cluster."
+
+    newvalues(%r{\d+}, :absent)
+
+    defaultto :absent
   end
   newproperty(:clone_node_max) do
     desc "How many copies of the resource can be started on a single node.
     Defaults to 1."
+
+    newvalues(%r{\d+}, :absent)
+
+    defaultto :absent
   end
 
   newproperty(:notify_clones) do
     desc "When stopping or starting a copy of the clone, tell all the other copies beforehand
       and when the action was successful.
       Allowed values: true, false"
```

Comment:
absent is more an internal thing, it probably does not need to be exposed in docs